client.go

```diff
@@ -170,8 +170,14 @@ type Config struct {
 	// HttpClient is the client to use. Default will be used if not provided.
 	HttpClient *http.Client `json:"-"`
 
-	// Routing mode
+	// Routing mode, one of "", "tenant" or "server".
 	RoutingMode string `json:"routing_mode"`
```

Comment:
I think that this config variable might be easier to understand, if the name was tied to vanity domains in some way.
Perhaps `VanityDomainType`, or something similar, would be better?

Reply:
OK, i'm going to rename RoutingMode -> VanityDomainType.